packages/field-plugin/src/createFieldPlugin/createPluginActions/partialPluginStateFromStateChangeMessage.ts

```diff
@@ -7,16 +7,28 @@ import {
 
 export const pluginStateFromStateChangeMessage = <Content>(
   message: LoadedMessage | StateChangedMessage,
-  parseContent: (content: unknown) => Content,
-): FieldPluginData<Content> => ({
-  spaceId: message.spaceId ?? undefined,
-  story: message.story ?? undefined,
-  storyId: message.storyId ?? undefined,
-  storyLang: message.language === '' ? 'default' : message.language,
-  blockUid: message.blockId ?? undefined,
-  token: message.token ?? undefined,
-  options: recordFromFieldPluginOptions(message.schema.options),
-  uid: message.uid ?? undefined,
-  content: parseContent(message.model),
-  isModalOpen: message.isModalOpen,
-})
+  validateContent: (content: unknown) => {
+    content: Content
+    error?: string
+  },
+): FieldPluginData<Content> => {
+  const validateResult = validateContent(message.model)
+  if ('error' in validateResult) {
+    console.warn(
```

Comment:
should we maybe also log the error message? It might be helpful for the developer to see which error message has been sent.